workload/s3/s3api/object_operations_workload_type3.yaml

```diff
@@ -0,0 +1,18 @@
+test_1:
+    TEST_ID: TEST-40042
+    object_size:
+        - 128Mb
+        - 256Mb
+        - 512Mb
+        - 1Gb
+        - 2Gb
+    write_percentage: 95  # Percentage of data to fill the storage.
+    read_percentage: 100  # Percentage of written data to be used to read from storage.
+    delete_percentage: 0  # Percentage of data to be deleted from storage.
+    cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+    #  If None then it will be fetched from cluster(cortx specific.)
+    total_storage_size: None  # Total available storage from cluster or user to be used.
+    min_runtime: 30d  # Minimum execution durations to mark workload pass.
+    sessions_per_node: 20  # Number of sessions per node.
```

Comment:
Lets change this to 2 sessions per node.

Reply:
done